### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by gld1982ltd**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -15,8 +15,7 @@
   ],
   "homepage": "https://github.com/skeletonframework/skeletonframework",
   "ignore": [
-    "**/.*",
-    "**/*",
+    "*",
```
Comment (benhinchley) [inline bower.json]: `*` will just ignore everything

File: bower.json [hunk 2]
```diff
@@ -15,8 +15,7 @@
...
     "!dist/*"
```
Comment (benhinchley) [inline bower.json]: does this rule work?? I am not 100% sure about how bower ignores function

**Comment by gld1982ltd** [comment]
Bower ignores just like git. https://github.com/bower/spec/blob/master/json.md#ignore

**Comment by benhinchley** [comment]
okay cool, just wasn't 100% sure :) :+1:

**Pull request merged**